### PR TITLE
added favicon and fixed SignIn issue

### PR DESCRIPTION
**Pull request by Subham142**
## Related Issues
added favicon and title and also fixed the sign-in link on the SignUp page
Closes: #70

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

Screenshot (38)

### CODE REVIEW

File: client/src/Component/SignUp/SignUp.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ const SignUp = ()=>{
                     <button 
                     className="create_account_button"
                     onClick={()=>{PostData()}}>Create account</button>
-                    <p className="signup_signin_message">Already a user? <span className="signin_link">Sign in</span></p>
+                    <p className="signup_signin_message">Already a user? <a href="/signin" className="signin_link">Sign In</a></p>
```
Comment (maw1a) [inline client/src/Component/SignUp/SignUp.js:154]: Replace the <a> tag with a <Link> component from react-router-dom.

File: client/public/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="images/favicon.png" />
```
Comment (maw1a) [inline client/public/index.html:5]: This shouldn't work. 
favicon isn't added to the assets yet. If you want to add a logo u can use /images/logo.png for now

**Review by maw1a — CHANGES_REQUESTED** [review]
I have mentioned whatever changes are required.
Do make the required changes

File: client/src/Component/SignUp/SignUp.js (hunk 1 above)
Comment (Subham142) [reply client/src/Component/SignUp/SignUp.js:154]: I thought of doing that but the SignIn page <a> tag was used  therefore I thought not to break the flow

Comment (Subham142) [reply client/src/Component/SignUp/SignUp.js:154]: So  should i change he both pages?

Comment (Subham142) [reply client/src/Component/SignUp/SignUp.js:154]: @purnima143  please confirm

File: client/public/index.html (hunk 2 above)
Comment (Subham142) [reply client/public/index.html:5]: I made a copy of the logo as a favicon.png so that it won't get messed up

**Comment by maw1a** [comment]
@purnima143 Can you resolve this?

**Comment by Subham142** [comment]
@maw1a  i'll make given  the changes today. My semester exams are going on that's why the delay

**Comment by maw1a** [comment]
> @maw1a i'll make given the changes today. My semester exams are going on that's why the delay

Sure, take your time 👍

**Comment by Subham142** [comment]
@maw1a  Please review the changes

File: client/src/index.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './bootstrap.min.css'
```
Comment (maw1a) [inline client/src/index.js:3]: Bootstrap isn't being used in this project.
Remove bootstrap css.

**Review by maw1a — CHANGES_REQUESTED** [review]
Good work 👍
Can you remove bootstrap css and related imports

File: client/src/index.js (hunk 3 above)
Comment (Subham142) [reply client/src/index.js]: @maw1a  done

**Review by maw1a — APPROVED** [review]

**Pull request merged**